### PR TITLE
move avatar output (i.e., configurable machine) behind log flag

### PR DESCRIPTION
**Pull request by mariusmue**
In case you ever used avatar's configurable machine and where annoyed by all the logging output of created memory regions and similar, this PR provides the solution!
These prints are now guarded by qemu's logging/log_mask infrastructure. This required to adjust the type of the qemu log mask from int to long - but this change should be backwards compatible either way.

### CODE REVIEW

File: include/qemu/log.h [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 /* Private global variables, don't use */
 extern FILE *qemu_logfile;
-extern int qemu_loglevel;
+extern long qemu_loglevel;
```
Comment (lacraig2) [inline include/qemu/log.h]: Do we need to change this to a long? There's definitely still room in the `int` for an AVATAR log.

P.S. PR #1114 will add a `LOG_PANDA` so it'd be good to not use 17. 18-27 are available though!

**Review by lacraig2 — CHANGES_REQUESTED** [review]
Everything else looks good. Not changing to a `long` would simplify this significantly.

**Comment by lacraig2** [comment]
I merged that other old PR and fixed the conflict with this PR.

File: include/qemu/log.h (hunk 1 above)
Comment (mariusmue) [reply include/qemu/log.h]: Good catch for 18-27, I didn't realize and just appended after the last #define! Updated that branch accordingly

**Comment by mariusmue** [comment]
Uhm. I think the current CI failures are not related to my log_mask changes - how would we move forward here?

**Comment by lacraig2** [comment]
You're right. I'll run it again.

**Comment by lacraig2** [comment]
I went ahead and added this log mechanism to `qemu_logging.py` in pypanda. That'll let you selectively turn on/off various log types dynamically.

Once tests pass this should be good to go. Thanks for the PR!

**Review by lacraig2 — APPROVED** [review]

**Pull request merged**